funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_manager.py

```diff
@@ -561,7 +574,6 @@ def cli_run():
         pass
 
     try:
-        global logger
```

Comment:
We can't remove this line, because if we don't set the global variable logger to the logger created in the cli_run, the manager init will not see the value, and will try to recreate it.